### ISSUE TITLE
Identify and pin the minimum supported versions

### ISSUE THREAD

**Issue by maximlt** [holoviz/hvplot#1169]
This is certainly going to be a tedious task, but the minimum pins of the packages hvPlot directly depends on (e.g. Panel, HoloViews) and optionally depends on (e.g. GeoViews, Datashader) should be identified and updated in `setup.py` and in the documentation (in particular for the optional dependencies).

**Comment by Azaya89** [comment]
This feels more like an `enhancement` than a bug???

**Comment by maximlt** [comment]
Yes true. I think I labeled that as Bug at the time, as the pins were very old and hvPlot was very likely not working if installed with these old dependencies. They've been updated a bit since then.
